docs/configuration.md

```diff
@@ -187,12 +186,6 @@ Use this parameter to change the background color of the header. By default, the
 
 ---
 
-**styling.footerText** (optional)
```

Comment:
Do we not need to document the ENV value somewhere?

Reply:
Obviously not in this file. Maybe a task for next sprint to fill in the `.env` file some more with disabled examples

Reply:
Yes we should! I wanted to discuss this in the next sprint planning. I added to this document yesterday: https://videodock.atlassian.net/wiki/spaces/OTT/pages/2577858562/OTT+Sprint+2024-3

> file some more with disabled examples

Also good idea.  Good intermediate solution if you ask me.